iot-e2e-tests/edge-e2e/src/main/java/io/swagger/server/api/verticle/DeviceApiVerticle.java

```diff
@@ -11,6 +11,7 @@
 import io.swagger.server.api.MainApiException;
 import io.swagger.server.api.model.RoundtripMethodCallBody;
 
+@SuppressWarnings("unused")
```

Comment:
Nit: If we don't plan to address for tests later, I can see this to be "ALL" as well.